diff -rupN vmdebootstrap-0.5/vmdebootstrap vmdebootstrap-0.5.new/vmdebootstrap
--- vmdebootstrap-0.5/vmdebootstrap	2014-11-16 16:10:24.000000000 +0100
+++ vmdebootstrap-0.5.new/vmdebootstrap	2016-09-13 15:10:35.216648599 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 # Copyright 2011-2013  Lars Wirzenius
 # Copyright 2012  Codethink Limited
 # Copyright 2014 Neil Williams <codehelp@debian.org>
